packages/api/lambdas/kinesis-consumer.js

```diff
@@ -75,25 +80,86 @@ function validateMessage(event) {
   return validate(event);
 }
 
+
+/**
+ * Publish a record to the fallback SNS topic for further processing.
+ *
+ * @param {Object} record - errored record
+ * @returns {Promise<Object>} - SNS publish response
+ */
+async function publishRecordToFallbackTopic(record) {
+  log.debug('publishing bad record to ', fallbackArn);
```

Comment:
I would probably bring this up to `info` level as opposed to debug. Seems like something we'd want to see in the logs.